lms/static/js/student_account/views/LoginView.js

```diff
@@ -29,13 +30,20 @@
                 this.errorMessage = data.thirdPartyAuth.errorMessage || '';
                 this.platformName = data.platformName;
                 this.resetModel = data.resetModel;
+                this.supportURL = data.supportURL;
 
                 this.listenTo( this.model, 'sync', this.saveSuccess );
                 this.listenTo( this.resetModel, 'sync', this.resetEmail );
             },
 
             render: function( html ) {
                 var fields = html || '';
+                var message = 'We have sent an email message with password reset instructions to the email ' +
```

Comment:
``HtmlUtils.interpolateHtml`` doesn't do i18n, so you still need to use ``gettext``. Note also that you then cannot break the string onto multiple lines because of how the i18n text scraper works.